.changeset/shy-boxes-relate.md

```diff
@@ -0,0 +1,24 @@
+---
+'@primer/react-brand': minor
+---
+
+New CTABanner component available
+
+```jsx
+<CTABanner>
+  <CTABanner.Heading>
+    Where the most ambitious teams build great things
+  </CTABanner.Heading>
+  <CTABanner.Description>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+    ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+    elementum.
+  </CTABanner.Description>
+  <CTABanner.ButtonGroup>
+    <Button>Primary Action</Button>
+    <Button>Secondary Action</Button>
+  </CTABanner.ButtonGroup>
+</CTABanner>
+```
+
+:link: [See the documentation for more details and usage examples.](https://primer.style/brand/components/ComparisonTable)
```

Comment:
😅 

```suggestion
:link: [See the documentation for more details and usage examples.](https://primer.style/brand/components/CTABanner)
```

Reply:
Weird I fixed this when I updated the PR and it didn't commit.